doc/api.rst

```diff
@@ -28,4 +28,13 @@ Classifiers
     mapclassify.Percentiles
     mapclassify.Std_Mean
     mapclassify.User_Defined
+
+Utilities
+---------
+
+.. autosummary::
+   :toctree: generated/
+
+    mapclassify.classifiers.K_classifiers
```

Comment:
Since `gadf` and `K_classifiers` are added to `__init__.py`, the api here could be `mapclassify.gadf` and `mapclassify.K_classifiers`?